src/mlpack/tests/ann_dist_test.cpp

```diff
@@ -76,7 +76,8 @@ TEST_CASE("JacobianBernoulliDistributionTest", "[ANNDistTest]")
     }
 
     module.LogProbBackward(target, jacobianB);
-    REQUIRE(arma::max(arma::max(arma::abs(jacobianA - jacobianB))) <= 1e-5);
+      REQUIRE(arma::approx_equal(jacobianA, jacobianB,"both", 1e-5,1e-5));
```

Comment:
I don't understand your indentation changes. Am I missing something?

Reply:
Also, be consistent with spaces after commas.

Reply:
Yes, sorry that was mistake on my part. I will review and make necessary changes